src/main/java/com/minecolonies/colony/buildings/AbstractBuilding.java

```diff
@@ -147,23 +154,25 @@ private static void addMapping(String name, Class<? extends AbstractBuilding> bu
     /**
      * Checks if a block matches the current object.
      *
-     * @param block     Block you want to know whether it matches this class or not
-     * @return          True if the block matches this class, otherwise false
+     * @param block
+     *            Block you want to know whether it matches this class or not
+     * @return True if the block matches this class, otherwise false
      */
     public boolean isMatchingBlock(Block block)
     {
         Class<?> c = blockClassToBuildingClassMap.get(block.getClass());
         return getClass().equals(c);
     }
 
-
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)